lib/components/src/blocks/Preview.tsx

```diff
@@ -154,7 +156,11 @@ const Preview: FunctionComponent<PreviewProps> = ({
   }
   const showToolbar = withToolbar && !Array.isArray(children);
   return (
-    <PreviewContainer {...{ withSource, withToolbar: showToolbar }} {...props}>
+    <PreviewContainer
+      {...{ withSource, withToolbar: showToolbar }}
+      {...props}
+      className="sbdocs-preview"
```

Comment:
I think this should accept a className from props and merge with `"sbdocs-preview"` if present?